### PR TITLE
Announce MirageOS version 3.0.0

### PR DESCRIPTION
**Pull request by yomimono**

### CODE REVIEW

File: tmpl/blog/announcing-mirage-30-release.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+We're excited to announce MirageOS 3.0! MirageOS is a modern, modular library operating system that allows the creation of small, secure, legacy-free services. MirageOS applications can be compiled to run as self-contained virtual machines (a few MB in size) on Xen, KVM hosts FreeBSD's bhyve, or even as regular Unix processes (allowing access to regular debugging tools).  The system libraries themselves can be reused in traditiional applications, just like any other software library.
```
Comment (avsm) [inline tmpl/blog/announcing-mirage-30-release.md]: s/traditiional/traditional

File: tmpl/blog/announcing-mirage-30-release.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+### Disaggregated Module Type Definitions
+
+Breaking all of the MirageOS 3.0 APIs showed us that keeping them all in the same place made updates really difficult.  There's now an additional set of packages which contain the definitions for each set of module types (e.g. [mirage-fs](https://github.com/mirage/mirage-fs) for the `FS` module type, [mirage-block](https://github.com/mirage/mirage-block) for the `BLOCK` module type, etc).  A few module types had some additional useful code that was nicely functorized over the module type in question, so we've bundled that code in the module type packages as well.  Documentation for all of the module type packages is available at [the Mirage documentation hub](https://mirage.github.io/doc/).
```
Comment (avsm) [inline tmpl/blog/announcing-mirage-30-release.md]: url should be docs.mirage.io

**Comment by avsm** [comment]
@yomimono i pushed a couple of minor tweaks to note odoc is still beta and fix a typo

**Comment by yomimono** [comment]
thanks @avsm :)

**Pull request merged**